gnuradio-runtime/include/gnuradio/xoroshiro-variates.h

```diff
@@ -0,0 +1,139 @@
+/*
+  Copyright (C) 2017 Marcus Müller
```

Comment:
Did you copy this from another of your projects?

Reply:
yes.

Um, stupid question, so I have a CLA that says that code contributed to the GR project is inherently copyright-assigned to the FSF. So, no matter what, I'll have to change this line.

But: my other project is LGPL; I'd rather not dual-license it, but then again my guess is that I can't just "work around" the fact that GNU Radio is GPL by first having the same code in a toy python module elsewhere, right?

Reply:
If this is all your code, you can simply assign everything to FSF same with any other code you contribute to GNU Radio. That doesn't affect your own projects.